src/test/java/pl/byczazagroda/trackexpensesappbackend/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.*;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.byczazagroda.trackexpensesappbackend.dto.AuthRegisterDTO;
+import pl.byczazagroda.trackexpensesappbackend.model.User;
+import pl.byczazagroda.trackexpensesappbackend.repository.UserRepository;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.any;
+import org.mockito.ArgumentCaptor;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class UserServiceImplTest {
+    @Mock
```

Comment:
Code formating: Enter between class proprties and methods format the whole class

Reply:
still not resolved